Dapper.SqlBuilder/SqlBuilder.cs

```diff
@@ -106,7 +104,7 @@ public Template AddTemplate(string sql, dynamic parameters = null)
             return new Template(this, sql, parameters);
         }
 
-        void AddClause(string name, string sql, object parameters, string joiner, string prefix = "", string postfix = "", bool IsInclusive = false)
+        void AddClause(string name, string sql, object parameters, string joiner, string prefix = "", string postfix = "", bool isInclusive = false)
```

Comment:
this is fine, but just to add: if this was a `public` method we would have to retain the atypical name; it is fine to rename for non-`public`